bitbots_vision/bitbots_vision/vision_modules/yoeo_handler.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, config: Dict, model_directory: str):
         self._det_candidates: Dict = defaultdict(list)
         self._det_class_names: Union[None, List[str]] = None
 
-        self._image = None  # : Union[None, numpy.ndarray]
+        self._image: Union[None, np.ndarray] = None
```

Comment:
Did you test this?

Reply:
As the vision worked and did not unexpectedly die on the robot, I guess, this also worked.